Source/WebCore/rendering/MarkedText.h

```diff
@@ -37,7 +37,7 @@ struct TextBoxSelectableRange;
 
 struct MarkedText {
     // Sorted by paint order
-    enum Type {
+    enum class Type {
```

Comment:
a backing type like uint8_t should be specified

Reply:
Thanks, fixed in https://github.com/WebKit/WebKit/commit/f38ed60829e3f706aaa242289f6345f82931c107